### ISSUE TITLE
arangors 0.5.0 phantom compile errors

### ISSUE THREAD

**Issue by velvia** [fMeow/arangors#85]
I'm using rustc 1.57.0-nightly (f03eb6bef 2021-10-02).  Getting compile errors from 0.5.0 crate:

```
    Checking arangors v0.5.0
error[E0560]: struct `AqlQueryBuilder<'_, (__query, (HashMap<&str, Value>,), __count, __batch_size, __cache, __memory_limit, __ttl, __options)>` has no field named `_phantom`
   --> /Users/evan/.cargo/registry/src/github.com-1ecc6299db9ec823/arangors-0.5.0/src/aql.rs:153:13
    |
153 |             _phantom: self._phantom,
    |             ^^^^^^^^ help: a field with a similar name exists: `phantom`

error[E0609]: no field `_phantom` on type `AqlQueryBuilder<'a, (__query, (), __count, __batch_size, __cache, __memory_limit, __ttl, __options)>`
   --> /Users/evan/.cargo/registry/src/github.com-1ecc6299db9ec823/arangors-0.5.0/src/aql.rs:153:28
    |
153 |             _phantom: self._phantom,
    |                            ^^^^^^^^ help: a field with a similar name exists: `phantom`

Some errors have detailed explanations: E0560, E0609.
For more information about an error, try `rustc --explain E0560`.
```

**Comment by velvia** [comment]
Note that I see the code in the repo has been fixed and if I pull from this repo instead of 0.5.0 it compiles.  So there is something wrong with the 0.5.0 tag in crates.io, I think.

**Comment by inzanez** [comment]
Same here

**Comment by inzanez** [comment]
Ok. It seems that the current `master` branch is indeed correct, but the currently released 0.5 crate is not the same. Could you publish a fix?

**Comment by ManevilleF** [comment]
I fixed it in https://github.com/fMeow/arangors/pull/80 but it was not released

**Comment by inzanez** [comment]
Is there somebody else that could release it? Or,...is there any reason keeping a non-building version as the main released one? Or did that currently released 0.5 build with an older compiler?

**Comment by ManevilleF** [comment]
@fMeow is the only maintainer I think.

Since I could not wait for my [aragog](https://crates.io/crates/aragog) library I forked `arangors` and published a simplified [arangors_lite](https://crates.io/crates/arangors_lite).

you may use it, the only difference is that it compiles and I removed the `uclient` stuff only to keep `reqwest` and a cleaner features system

**Comment by mrtnzlml** [comment]
I was in the same situation and also forked `arangors` (privately). And I also removed the `uclient` stuff. 😅 Related issue: https://github.com/fMeow/arangors/issues/77 Seems like `arangors_lite` could be a good option. 👍

**Comment by ManevilleF** [comment]
@mrtnzlml You should checkout `aragog`, if you want a mapped CRUD API for your database

**Comment by Anderssorby** [comment]
I ended up publishing a new crate with this fix here https://crates.io/crates/arango_rs.

**Comment by ManevilleF** [comment]
> I ended up publishing a new crate with this fix here https://crates.io/crates/arango_rs.

There is also https://crates.io/crates/arangors_lite

**Comment by fMeow** [comment]
fixed in v0.5.1
